SU-SWS/jumpstart_ui
# D8CORE-000: Kernel Tests For All The Layouts

# READY FOR REVIEW 

# Summary
- Fixes bug where you get a 502 error on empty regions. 

# Needed By (Date)
- 1.0.1 release

# Urgency
- Med/High

# Steps to Test

1. Create a new build of acsf.cardinalsites with the stanford_profile with this branch
2. Edit a full page layout (home) and create a few empty sections with 1, 2, and 3 column templates
3. Save and review the page. You should not have PHP errors.
4. Edit the full page layout again and adjust the settings to center and other alignment options in the emtpy regions
5. Save and review the page without errors
6. Edit the page again and add content to each of the empty sections. 
7. Save and view the page. Assert the configuration options are working.

Full disclosure I have not tested this code and edited it directly on github.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
